Wrapper.php

```diff
@@ -361,12 +363,11 @@ public function stream_write($data)
      */
     public function dir_closedir()
     {
-        if (null === $handle = closedir($this->getStream())) {
-            $this->_stream     = null;
-            $this->_streamName = null;
-        }
+        closedir($this->getStream());
+        $this->_stream     = null;
```

Comment:
http://php.net/manual/en/function.closedir.php

`closedir` always return `void`, so the `if` is always true.